packages/lib/src/model.ts

```diff
@@ -11,7 +11,9 @@ export enum ModelID {
   Claude_V1_100k = "anthropic/claude-v1-100k",
   // Other
   Together = "togethercomputer/GPT-NeoXT-Chat-Base-20B",
-  Cohere = "cohere/command-nightly"
+  Cohere = "cohere/command-nightly",
+  // Media Models
+  Shap_e = "openai/shap-e",
```

Comment:
Let's create two types of ModelIDs and unify them in one object, so that we can typecheck them separately. e.g. `TextModelID` and `MediaModelID`. Example of unification:

```ts
export const OpenAIModel = {
  GPT_3: ModelID.GPT_3,
  GPT_4: ModelID.GPT_4
} as const;
export type OpenAIModel = (typeof OpenAIModel)[keyof typeof OpenAIModel];

export const AnthropicModel = {
  Claude_Instant_V1: ModelID.Claude_Instant_V1,
  Claude_Instant_V1_100k: ModelID.Claude_Instant_V1_100k,
  Claude_V1: ModelID.Claude_V1,
  Claude_V1_100k: ModelID.Claude_V1_100k
} as const;
export type AnthropicModel =
  (typeof AnthropicModel)[keyof typeof AnthropicModel];

export const Model = {
  ...OpenAIModel,
  ...AnthropicModel
} as const;
export type Model = (typeof Model)[keyof typeof Model];
```

Reply:
Updated to combine two different enums called MediaModelID & TextModelID.
<img width="484" alt="image" src="https://github.com/alexanderatallah/window.ai/assets/107221725/773eaf96-2900-4091-a9d5-cfb2cc4f3ac4">